### PR TITLE
Cleanup factory-registration security rm page

### PR DESCRIPTION
**Pull request by kprosise**
Style and readability fixes applied.

The subsections had duplicate naming which I attempted to address accurately.

Viewed rendered output. Ran linter.

This commit applies to FFTK-2795

## Readiness

*   [x] Merge (pending reviews)
*   [ ] Merge after _date or event_
*   [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

*   [x] Run spelling and grammar check, preferably with linter.
*   [ ] Avoid changing any header associated with a link/reference.
*   [ ] Step through instructions (or ask someone to do so).
*   [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
*   [x] Match tone and style of page/section.
*   [x] Run `make linkcheck`.
*   [x] View HTML in a browser to check rendering.
*   [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
*   [x] follow best practices for commits.
    *   [x] Descriptive title written in the imperative.
    *   [x] Include brief overview of QA steps taken.
    *   [x] Mention any related issues numbers.
    *   [x] End message with sign off/DCO line (`-s, --signoff`).
    *   [x] Sign commit with your gpg key (`-S, --gpg-sign`).
    *   [ ] Squash commits if needed.
*   [ ] Request PR review by a technical writer and at least one peer.

## Comments

Section headers/titles should be checked for accuracy.

### CODE REVIEW

**Comment by doanac** [comment]
Docs for e3eb5cd9469d399bcc689b245825718b870c3ce8 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2545/docs/artifacts/html/index.html

**Review by vanmaegima — APPROVED** [review]

File: source/reference-manual/security/factory-registration-ref.rst [hunk 1]
```diff
@@ -3,90 +3,73 @@
...
 
-This scenario is handy for certain security constrained setups. However,
-it does have a couple of potential drawbacks:
+In this scenario, devices connect to a reference server instance on a private network.
+This network is isolated from the internet (``api.foundries.io``).
+Devices get a valid signed client certificate from the reference server.
+Each device will be created **on-the-fly** via ``api.foundries.io`` the first time they connect.
+
+This is handy for certain security constrained setups.
+However, it does have a couple potential drawbacks:
```
Comment (StealthyCoder) [inline source/reference-manual/security/factory-registration-ref.rst:30]: I am being pedantic here, I do not consider them drawbacks or potential drawbacks. They are just notes/caveats that are part of this solution. It is also nice to know devices do not populate unless they have for sure connected, which also gives some insight into how many devices sold vs how many connected for example.

**Review by StealthyCoder — APPROVED** [review]

File: source/reference-manual/security/factory-registration-ref.rst (hunk 1 above)
Comment (kprosise) [inline source/reference-manual/security/factory-registration-ref.rst:30]:
```suggestion
However, it does have a couple caveats:
```

**Comment by doanac** [comment]
Docs for 96d5453109eba9468a53d3cc0665b94e1fff34d6 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2549/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 3583a01649572168f9d50310a554aed0de509d0a are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2550/docs/artifacts/html/index.html

**Pull request merged**